### PR TITLE
[WIP]Enable debuggertests win

### PR DESCRIPTION
**Pull request by alsemenn**
Temporary disabled failing debugger tests.
Need to add option to skip debugger tests.

### CODE REVIEW

**Comment by danmosemsft** [comment]
@alsemenn is this still in progress? we should finish or close this PR.

**Comment by alsemenn** [comment]
This is still in progress.

File: tests/debugger_tests/ConfigFilesGenerators/ConfigTemplate.Windows_NT.x64.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ <DebuggeeRootDir>$(TestRoot)\Debuggees</DebuggeeRootDir> 
+ <DebuggeeSourceRoot>$(DebuggeeRootDir)\Source</DebuggeeSourceRoot>
+ <RuntimeRoot>##Insert_Runtime_Root##</RuntimeRoot>
+ <NugetCacheRoot>##Insert_Nuget_Cache_Root##</NugetCacheRoot>
+ <TestProduct>ProjectK</TestProduct> 
+ <HostExe>$(RuntimeRoot)\corerun.exe</HostExe> 
+ <DbgShim>$(RuntimeRoot)\dbgshim.dll</DbgShim> 
+ <RuntimeSymbolsPath>$(RuntimeRoot)</RuntimeSymbolsPath> 
+ <DebuggeeBuildProcess>cli</DebuggeeBuildProcess> 
+ <CliVersion>1.0.0-preview1-002702</CliVersion>
```
Comment (mikem8361) [inline tests/debugger_tests/ConfigFilesGenerators/ConfigTemplate.Windows_NT.x64.txt]: This CliVersion is a lot older than the one in the debuggertests repo's config files (1.0.0-preview2.1-003155).  Is this used?

File: tests/debugger_tests/ConfigFilesGenerators/ConfigTemplate.Windows_NT.x64.txt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ <NugetCacheRoot>##Insert_Nuget_Cache_Root##</NugetCacheRoot>
+ <TestProduct>ProjectK</TestProduct> 
+ <HostExe>$(RuntimeRoot)\corerun.exe</HostExe> 
+ <DbgShim>$(RuntimeRoot)\dbgshim.dll</DbgShim> 
+ <RuntimeSymbolsPath>$(RuntimeRoot)</RuntimeSymbolsPath> 
+ <DebuggeeBuildProcess>cli</DebuggeeBuildProcess> 
+ <CliVersion>1.0.0-preview1-002702</CliVersion> 
+ <DebuggeeNativeLibRoot>$(DebuggeeRootDir)\native</DebuggeeNativeLibRoot> 
+ <DebuggeeBuildRoot>$(DebuggeeRootDir)\dotnet.$(CliVersion)</DebuggeeBuildRoot> 
+ <ProjectJsonNetStandardLibraryVersion>1.0.0-rc2-23811</ProjectJsonNetStandardLibraryVersion>
```
Comment (mikem8361) [inline tests/debugger_tests/ConfigFilesGenerators/ConfigTemplate.Windows_NT.x64.txt]: ProjectJsonNetStandLibraryVersion, ProjectJsonSystemThreadingThreadVersion and ProjectJsonMirosoftCSharpVersion don't exists in the debuggertests repo's config files.

File: tests/debugger_tests/setup-debuggertests.cmd [hunk 3]
```diff
@@ -51,7 +66,9 @@ REM ===
 REM === download debuggertests package
 REM ===
 REM =========================================================================================
-set DEBUGGERTESTS_URL=https://dotnetbuilddrops.blob.core.windows.net/debugger-container/Windows.DebuggerTests.zip
+REM TODO: change the archive name!!! (this is a test-only)
+REM TODO: name should be built from __OS and __Arch parameters.
```
Comment (noahfalk) [inline tests/debugger_tests/setup-debuggertests.cmd]: Just marking this todo with a comment for myself so I can confirm the final version resolved it

**Comment by alsemenn** [comment]
Noah's comments:
1 and 2 - will do
3 - There is a way to disable single test (using coreclr infrastructure): by modifying tests/issues.targets  (the same way I've disabled failing tests)

**Comment by noahfalk** [comment]
For disabling tests my comment was to ensure there is documentation that tells someone how to disable a test. If that also already exists then nothing to be done.

**Comment by alsemenn** [comment]
Going to use another approach.

**Pull request closed without merging**